wonnx-cli/src/util.rs

```diff
@@ -0,0 +1,205 @@
+use image::imageops::FilterType;
```

Comment:
I think that this file should be called `utils` for utilities.

Reply:
OK, easy fix :-)